db/schema.rb

```diff
@@ -326,6 +326,21 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "whitehall_imported_assets", force: :cascade do |t|
+    t.bigint "whitehall_import_id", null: false
+    t.bigint "file_attachment_revision_id"
+    t.bigint "image_revision_id"
+    t.string "original_asset_url", null: false
+    t.string "variants", default: "{}"
+    t.string "state", default: "not_processed", null: false
+    t.text "error_log"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
```

Comment:
Oh these precisions are new, I assume this is something Rails did automatically?

They seem great though as I think that'll mean Ruby and postgres agree on precision of time which might mean this code works in tests:

```
time = Time.current
m = Model.create(time: time)
expect(m.time).to eq(time)
```

Reply:
Yep, this came for free with Rails 6 👍